src/cmake/Dependencies.cmake

```diff
@@ -973,6 +973,7 @@ if(AV_BUILD_OPENCV)
         CONFIGURE_COMMAND 
             ${CMAKE_COMMAND} 
             ${CMAKE_CORE_BUILD_FLAGS}
+            -DCMAKE_PREFIX_PATH=${CMAKE_INSTALL_PREFIX}
```

Comment:
Maybe better to add a variable
`set(FFMPEG_CMAKE_FLAGS -DFFMPEG_ROOT=${CMAKE_INSTALL_PREFIX})`
And add it bellow as it's done for the other libs "${TIFF_CMAKE_FLAGS} ${PNG_CMAKE_FLAGS} ${JPEG_CMAKE_FLAGS} ${LIBRAW_CMAKE_FLAGS}"